.cirrus.yml

```diff
@@ -380,21 +385,30 @@ testing_task:
         $CIRRUS_CHANGE_MESSAGE !=~ '.*CI:IMG.*' &&
         $CIRRUS_CHANGE_MESSAGE !=~ '.*CI:DOCS.*'
 
-    gce_instance:
-        matrix:
-            image_name: "${FEDORA_CACHE_IMAGE_NAME}"
-            image_name: "${PRIOR_FEDORA_CACHE_IMAGE_NAME}"
-            # Multiple test failures on Ubuntu 19 - Fixes TBD in future PR
-            # TODO: image_name: "${UBUNTU_CACHE_IMAGE_NAME}"
-            image_name: "${PRIOR_UBUNTU_CACHE_IMAGE_NAME}"
+    matrix:
+        - name: "testing ${FEDORA_NAME}"
+          gce_instance:
+              image_name: "${FEDORA_CACHE_IMAGE_NAME}"
```

Comment:
Okay now that is some black-magic voodoo right there!

I don't see that documented anywhere, and yet it makes a huge impact.  Wow!

I worry this might be an unintended feature (b/c it's undocumented?), it might be worth submitting a documentation update to see how the Cirrus people respond (and to update the docs for everybody's benefit).  I'd hate this feature break/disappear in the future.  

Reg: https://github.com/cirruslabs/cirrus-ci-docs/blob/master/docs/guide/writing-tasks.md

(Plus, they REALLY appreciate help with docs as they're all non-native English speakers.)

Reply:
I got the idea from an [already-closed issue](https://github.com/cirruslabs/cirrus-ci-docs/issues/278)

Reply:
Wow...that's some deep/clever searching.